src/options.cpp

```diff
@@ -4,6 +4,7 @@
 #include <climits>
 #include <clocale>
 #include <iterator>
+#include <limits>
```

Comment:
It is already included through `climits` above.

Reply:
IWYU failed to run twice, i am not sure is it an issue of this pr or not
ill restart it again, but i suspect it would fail this time either

Reply:
it's an issue with IWYU setup, not with this PR.
I'll take a look, but a bit later, I'm very busy this week.

Basically, if IWYU fails with no obvious "please fix your includes!!!" error - it's a setup issue and can be ignored/treated as passing.